src/Components/ImagesTable/Status.tsx

```diff
@@ -288,19 +288,27 @@ type ErrorStatusPropTypes = {
 
 const ErrorStatus = ({ icon, text, error }: ErrorStatusPropTypes) => {
   let reason = '';
+  const detailsArray: string[] = [];
   if (typeof error === 'string') {
     reason = error;
   } else {
     if (error.reason) {
       reason = error.reason;
     }
-    if (error.details?.reason) {
-      if (reason !== '') {
-        reason = `${reason}\n\n${error.details?.reason}`;
-      } else {
-        reason = error.details.reason;
+    if (Array.isArray(error.details)) {
+      for (const line in error.details) {
+        detailsArray.push(`${error.details[line]}`);
       }
     }
+    if (typeof error.details === 'string') {
+      detailsArray.push(error.details);
+    }
+    if (error.details?.reason) {
+      detailsArray.push(`${error.details.reason}`);
+    }
+    if (error.details?.s) {
```

Comment:
Huh what's the `s` field?

Reply:
Update from our chats aside this issue:
It seems to stem from my confusion from the golang error type having the error in `.s`

https://github.com/golang/go/blob/1843464f014c946c1663de76249267486887626f/src/errors/errors.go#L67

but this seems to never land in the frontend anyway due to
https://github.com/golang/go/blob/1843464f014c946c1663de76249267486887626f/src/builtin/builtin.go#L309

not being able to serialize …
I'll remove from the frontend...